hooks/post_gen_project.py

```diff
@@ -150,18 +175,4 @@ def validate_manifest():
    https://github.com/chanzuckerberg/napari-hub/blob/main/docs/customizing-plugin-listing.md
 """
 
-{% if cookiecutter.github_repository_url != 'provide later' %}
```

Comment:
I do not know why there is a check for `'provide later'`. I remove it, but maybe I do not know the scenario where it is needed.

Reply:
but it looks like #75 was wrong if introduced, and proper was introduced in #78? 
I still do not see why it depends on providing the repository.

Reply:
So it looks like in this moment it is enough to check if user ask for pre-commit